### PR TITLE
fix(orb-ui): #1110 #1144 fix sink control on dataset-from component.

### PR DESCRIPTION
**Pull request by gpazuch**
user was not able to add a sink that has been already added and removed.
user was not able to remove a sink.

### CODE REVIEW

File: ui/src/app/shared/components/orb/sink-control/sink-control.component.html [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
+  data-orb-qa-id="addSink"
+  ghost
+  nbButton>
+  <nb-icon
+    icon="plus-outline"
+    size="14"
+    status="primary"
+    style="color: #df316f;">
   </nb-icon>
 </button>
```
Comment (lpegoraro) [inline ui/src/app/shared/components/orb/sink-control/sink-control.component.html:27]: empty line at end of file, you want to get rid of this, set up this in your editor
Screenshot from 2022-05-04 11-33-22

File: ui/src/app/shared/components/orb/sink-display/sink-display.component.html [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
-               icon="close-outline"
-               size="12"></nb-icon>
+      <nb-icon
+        (click)="onRemoveSink(sink)"
+        class="ml-1"
+        icon="close-outline"
+        size="12"></nb-icon>
     </mat-chip>
   </mat-chip-list>
 </div>
```
Comment (lpegoraro) [inline ui/src/app/shared/components/orb/sink-display/sink-display.component.html:15]: empty line at end of file.

File: ui/src/app/shared/components/orb/sink-control/sink-control.component.html (hunk 1 above)
Comment (lpegoraro) [reply ui/src/app/shared/components/orb/sink-control/sink-control.component.html:27]: In vs code you can use in the settings.json too
```json
"files.trimTrailingWhitespace": true
"files.trimFinalNewlines": true
"files.insertFinalNewline": true
```

**Review by lpegoraro — APPROVED** [review]
Looks good, I have a few comments on a few standards for enhancing git diff evaluation.

File: ui/src/app/shared/components/orb/sink-control/sink-control.component.html (hunk 1 above)
Comment (gpazuch) [reply ui/src/app/shared/components/orb/sink-control/sink-control.component.html:27]: @lpegoraro interesting. I believe Intellij may have read this rule from existing codebase - mainflux.

I usually forget to leave those newlines in newly created files and get the complain, hence it is in my autoformat to leave one empty line.

I can update this rule and perhaps wait to apply it to all documents on the beginning of the next sprint.

File: ui/src/app/shared/components/orb/sink-display/sink-display.component.html (hunk 2 above)
Comment (gpazuch) [reply ui/src/app/shared/components/orb/sink-display/sink-display.component.html:15]: :+1:

**Pull request merged**